tests/test_accessibility.py

```diff
@@ -108,6 +112,196 @@ def test_axs_audit_limit_rules(self, mock_rules):
 
         self.assertEqual(0, len(result.warnings))
 
+    @patch('tests.pages.AccessibilityPage.axs_audit_rules_to_run')
+    def test_axs_audit_run_multiple_rules(self, mock_rules):
+        # Tests that other all errors are reported if checking more than one.
+        mock_rules.return_value = ['badAriaRole', 'badAriaAttributeValue']
+        page = AccessibilityPage(self.browser)
+        page.visit()
+        report = page.do_axs_audit()
+
+        mock_rules.assert_called_once_with()
+
+        # There was one page in this session
+        self.assertEqual(1, len(report))
+        result = report[0]
+
+        # It is expected that both rules will be reported.
+        self.assertEqual(2, len(result.errors))
+        self.assertIn(self.ax_aria_01_errors, result.errors)
+        self.assertIn(self.ax_aria_04_errors, result.errors)
+
+        self.assertEqual(0, len(result.warnings))
+
+    @patch('tests.pages.AccessibilityPage.axs_audit_rules_to_ignore')
+    def test_axs_audit_ignore_rule(self, mock_ignore):
+        # Ignore the rule AX_ARIA_01. For this test, that means the only rule
+        # to result in an error should be AX_ARIA_04.
+        mock_ignore.return_value = ['badAriaRole']
+        page = AccessibilityPage(self.browser)
+        page.visit()
+        report = page.do_axs_audit()
+
+        mock_ignore.assert_called_once_with()
+
+        # There was one page in this session
+        self.assertEqual(1, len(report))
+        result = report[0]
+
+        self.assertEqual(1, len(result.errors))
+        self.assertEqual(result.errors[0], self.ax_aria_04_errors)
+
+        self.assertEqual(0, len(result.warnings))
+
+    @patch('tests.pages.AccessibilityPage.axs_audit_rules_to_ignore')
+    def test_axs_audit_ignore_multiple_rules(self, mock_ignore):
+        # Ignore multiple rules. In this test, we are explicitly ignoring all.
```

Comment:
nit: I'd say "explicitly ignoring each" instead of "explicitly ignoring all." Just b/c when I first read it, I incorrectly think we have some "ignore_all_rules" capability...